handlers.go

```diff
@@ -2,14 +2,13 @@ package main
 
 import (
 	"errors"
-	"fmt"
 
 	"github.com/charmbracelet/x/ansi"
 )
 
 var csiHandlers = map[int]handlerFn{
 	'm': handleSgr,
-	'c': printf("Request primary device attributes"),
+	'c': printWithMnemonic("DA1", "Request primary device attributes"),
```

Comment:
This is unrelated but the description for DA1 should just be:
```suggestion
	'c': printWithMnemonic("DA1", "Primary device attributes"),
```
When there is no parameters `\x1b[c`, it means a request for the terminal to send its primary device attributes which then responds with something like `\x1b[64;1;2;3;4c`, the same sequence command but with paramters.